packages/e2e-tests/src/specs/editor/statusCheck.js

```diff
@@ -27,7 +27,8 @@ import {
  */
 import { addAllowedErrorMessage } from '../../config/bootstrap';
 
-describe('Status Check', () => {
+// eslint-disable-next-line jest/no-disabled-tests
+describe.skip('Status Check', () => {
```

Comment:
There might be something going on here with Network requests -- need to narrow that down.

Reply:
Let's create a ticket, updating the comment here to include the ticket number